zulipterminal/ui_tools/buttons.py

```diff
@@ -322,6 +322,8 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
         if is_command_key("TOGGLE_TOPIC", key):
             # Exit topic view
             self.view.left_panel.show_stream_view()
+        elif is_command_key("TOPIC_INFO", key):
```

Comment:
Probably better to move the hotkey changes here since `i` is not usable until this point.